code/modules/holiday/easter.dm

```diff
@@ -0,0 +1,242 @@
+//Easter start
+/datum/holiday/easter/greet()
+	return "Greetings! Have a Happy Easter and keep an eye out for Easter Bunnies!"
+
+/datum/round_event_control/easter
+	name = "Easter Eggselence"
+	holidayID = EASTER
+	typepath = /datum/round_event/easter
+	weight = -1
+	max_occurrences = 1
+	earliest_start = 0
+
+/datum/round_event/easter/announce()
+	priority_announce(pick("Hip-hop into Easter!","Find some Bunny's stash!","Today is National 'Hunt a Wabbit' Day.","Be kind, give Chocolate Eggs!"))
+
+
+/datum/round_event_control/rabbitrelease
+	name = "Release the Rabbits!"
+	typepath = /datum/round_event/rabbitrelease
+	weight = 5
+	max_occurrences = 10
+
+/datum/round_event/rabbitrelease/announce()
+	priority_announce("Unidentified furry objects detected coming aboard [station_name()]. Beware of Adorable-ness.", "Fluffy Alert", 'sound/AI/aliens.ogg')
+
+
+/datum/round_event/rabbitrelease/start()
+	for(var/obj/effect/landmark/R in landmarks_list)
+		if(R.name != "blobspawn")
+			if(prob(35))
+				if(istype(R.loc,/turf/space))
+					new /mob/living/simple_animal/chicken/rabbit/space(R.loc)
+				else
+					new /mob/living/simple_animal/chicken/rabbit(R.loc)
+
+/mob/living/simple_animal/chicken/rabbit

```

Comment:
I call them Rickens.. Or Cabbits. I'unno blame Nanotrasen and their crackpot genetics team.